.circleci/config.yml

```diff
@@ -1,98 +0,0 @@
-version: 2.1
```

Comment:
Why did we remove this file?

Reply:
@ajaybushel what about this question please?

Reply:
That is for Circle CI which we are not using. Even if we look forward to using a CI, it would either be DevOps or GitHub Actions. Unused file that could have been added with the original template used to bootstrap this project.

Reply:
sounds good, thanks for the clarification @ajaybushel